test/pkg/utils/utiles.go

```diff
@@ -0,0 +1,28 @@
+package utils
```

Comment:
typo in file name, why not storing it in the functional package and use it in main as well?